src/Commands/PowerPlatform/Environment/GetPowerPlatformSolution.cs

```diff
@@ -0,0 +1,65 @@
+﻿using PnP.PowerShell.Commands.Base;
+using PnP.PowerShell.Commands.Utilities.REST;
+using System;
+using System.Management.Automation;
+using System.Linq;
+using PnP.PowerShell.Commands.Base.PipeBinds;
+
+namespace PnP.PowerShell.Commands.PowerPlatform.Environment
+{
+    [Cmdlet(VerbsCommon.Get, "PnPPowerPlatformSolution")]
+    public class GetPowerPlatformSolution: PnPAzureManagementApiCmdlet
```

Comment:
Same here:

```
Would require this I think:

```
[RequiredMinimalApiPermissions("https://management.azure.com/.default")]
```
```

Reply:
For PowerPlatform related commands I don't think it would require...